ci-operator/step-registry/telcov10n/system/telco-kpis/collect-node-info/telcov10n-system-telco-kpis-collect-node-info-commands.sh

```
@@ -0,0 +1,182 @@
+    local bastion_host_vars="$3"
+
+    echo "Configuring SSH jump through hypervisor to reach bastion..."
+
+    local hypervisor_ip
+    hypervisor_ip=$(tr -d '[:space:]' < "${mounted_host_inventory}/common/hypervisor/ansible_host")
+
+    local ssh_user
+    ssh_user=$(tr -d '[:space:]' < "${mounted_group_inventory}/common/all/ansible_user")
+
+    local ssh_key_file="/tmp/ssh-jump-key"
+    [[ $- == *x* ]] && local was_tracing=true || local was_tracing=false
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)